hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/XceiverClientManager.java

```diff
@@ -168,7 +169,8 @@ private XceiverClientSpi acquireClient(Pipeline pipeline, boolean read)
       throws IOException {
     Preconditions.checkNotNull(pipeline);
     Preconditions.checkArgument(pipeline.getNodes() != null);
-    Preconditions.checkArgument(!pipeline.getNodes().isEmpty());
+    Preconditions.checkArgument(!pipeline.getNodes().isEmpty(),
+        FAILED_TO_FIND_HEALTHY_NODES);
```

Comment:
I think `NO_REPLICA_FOUND` would better reflect the situation.  There may be other, live datanodes, and "healthy nodes" seem more related to placement of new data.

Reply:
Thanks @adoroszlai for review, I change it.